cmd/commands/compose.go

```
@@ -59,11 +61,11 @@ func ConvertCommand(dockerCli command.Cli, options *composeOptions) *cobra.Comma
 				return err
 			}
 
-			j, err := template.JSON()
+			json, err := cloudformation.Marshall(template)
 			if err != nil {
 				fmt.Printf("Failed to generate JSON: %s\n", err)
```

Are we deliberately ignoring errors here? Should this error be returned?